rampwf/score_types/tests/test_soft_accuracy.py

```diff
@@ -0,0 +1,43 @@
+from __future__ import division
+
+import numpy as np
+
+from rampwf.score_types.soft_accuracy import SoftAccuracy
+
+
+score_matrix_1 = np.array([
+    [1, 0, 0],
+    [0, 1, 0],
+    [0, 0, 1],
+])
+
+score_matrix_2 = np.array([
+    [1, 0.5, 0],
+    [0.3, 1, 0.3],
+    [0, 0.5, 1],
+])
+
+y_true_proba_1 = np.array([1, 0, 0])
+y_true_proba_2 = np.array([0, 1, 0])
+y_true_proba_3 = np.array([0.5, 0.5, 0])
+
+y_proba_1 = np.array([1, 0, 0])
+y_proba_2 = np.array([0, 1, 0])
+
+
+def test_soft_accuracy():
+    score_1 = SoftAccuracy(score_matrix=score_matrix_1)
+    assert score_1(np.array([y_true_proba_1]), np.array([y_proba_1])) == 1
```

Comment:
no not of np.array they are already arrays

Reply:
[y_true_proba_1] would be just a list of np.arrays, no?